library/src/test/java/com/schibsted/spain/barista/rule/flaky/FlakyStatementBuilderTest.java

```diff
@@ -59,13 +69,36 @@ public void allowFlakyStatementReturnedWhenNoAnnotationsFoundButUsesDefault() th
     Statement baseStatement = new SomeStatement();
     Description description = Description.EMPTY;
 
-    Statement resultStatement = new FlakyStatementBuilder()
+    Statement resultStatement = createStatementWithAllowFlakyByDefault(baseStatement, description);
+
+    assertTrue(resultStatement instanceof AllowFlakyStatement);
+  }
+
+  @Test
+  public void lastStatementReturnedWhenRepeatAttemptsByDefaultAndAllowFlakyStatementUsedAtTheSameTime() throws Exception {
```

Comment:
Not that important because it's easier to get, but just to be coherent:

`lastStatementReturnedWhenDefaultRepeatAttemptsAndAllowFlakyStatementUsedAtTheSameTime`

Reply:
Changed 👍